WooCommerce/Classes/ViewRelated/Dashboard/Factories/StatsV4DataHelper.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import Yosemite
+import WooFoundation
+
+/// Helpers for calculating and displaying Stats V4 data
+///
+final class StatsV4DataHelper {
```

Comment:
This looks better suited for a `struct` type, but not a biggie!

Reply:
Also, how do you feel about naming it along the lines of `StatsDataTextFormatter`. 

Reason: Helper is quite a broad name, having a more concrete name limits the type of responsibility which makes it easy to maintain.

Also, Not a big deal 😅

Reply:
Good points! I'll change it!